starboard/android/shared/system_egl.cc

```diff
@@ -0,0 +1,146 @@
+// Copyright 2019 The Cobalt Authors. All Rights Reserved.
```

Comment:
Is this file refactored or moved from somewhere? I do not see any method in this interface being referenced for the fix.

Reply:
The change that activates this code is in starboard/android/shared/BUILD.gn and the operating different is the implementation of `SbGetEglInterface` that now returns a struct with the wrapped `eglInitialize`/`eglTerminate`/`eglMakeCurrent`.